packages/legend-query/src/components/QueryBuilderValueSpecificationEditor.tsx

```diff
@@ -250,6 +250,31 @@ export const DatePrimitiveInstanceValueEditor = observer(
   },
 );
 
+export const DateTimePrimitiveInstanceValueEditor = observer(
+  (props: {
+    valueSpecification: PrimitiveInstanceValue;
+    className?: string | undefined;
+  }) => {
+    const { valueSpecification, className } = props;
+    const value = valueSpecification.values[0] as string;
+    const changeValue: React.ChangeEventHandler<HTMLInputElement> = (event) => {
+      valueSpecification.changeValue(event.target.value, 0);
+    };
+
+    return (
+      <div className={clsx('query-builder-value-spec-editor', className)}>
+        <input
+          className="panel__content__form__section__input query-builder-value-spec-editor__input"
+          type="datetime-local"
```

Comment:
@MauricioUyaguari what should it be here? `local` or `UTC`?

Reply:
I think we should do `UTC`

Reply:
Could we try this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local#setting_timezones
so we set the timezone to UTC?

Reply:
Turns out `datetime-local` already assume `UTC` 😆 so we're good I guess